routes/payroll/controllers.js

```diff
@@ -0,0 +1,56 @@
+import PayrollService from "../../services/payroll.js";
+import httpResponse from "../../utils/httpResponse.js";
+
+const controller = {
```

Comment:
Why Payroll is included in User Pagination?

Reply:
Sir it is ok. it is not in the user controller it is the pay roll controller and the user pagination is in the user controller

Reply:
PR is named as User pagination...